delivery_postlogistics/postlogistics/web_service.py

```diff
@@ -455,9 +455,9 @@ def get_access_token(cls, picking_carrier):
 
     def _sanitize_string(self, value):
         """Removes disallowed chars ("|", "\", "<", ">", "’", "‘") from strings."""
-        if isinstance(value, str):
-            for char, repl in DISALLOWED_CHARS_MAPPING.items():
-                value = value.replace(char, repl)
+        value = value or ""
```

Comment:
Wouldn't it be a tiny bit prettier to set it as default in the parameter?

`def _sanitize_string(self, value=""):`

Reply:
The issue is that this method is called with `value=False` so even by setting a default value that won't work.